### DISCUSSION TITLE
How do I prevent AU from changing the version number?

### DISCUSSION THREAD

**Discussion by jharrilim** [chocolatey-community/chocolatey-au#27] [Q&A] [answered]
Currently when I try to push an update using AU and Appveyor, [AU sets the patch version to the entire date, and removes leading zeroes](https://github.com/jharrilim/chocolatey-packages/commit/812116a96c11c2ca3efa3be76a00536775eec877). `23.05` becomes `23.5.0.20230527`.

I'm not too sure how to prevent this from happening. I'd like for it not to because I always have to comment on the package on chocolatey saying this version isn't for review.

**Comment by TheCakeIsNaOH** [comment, marked as answer]
For AU, if force is enabled, then AU will update the package to use a fix version (aka adding the date to the end of the version), unless the desired package version is explicitly set. 

You are setting force always to be true when running `update-all`: https://github.com/jharrilim/chocolatey-packages/blob/812116a96c11c2ca3efa3be76a00536775eec877/update_all.ps1#L9

I would suggest setting it to be false normally, and you can explicitly enable force for specific Appveyor runs with the `[AU ......]` commit style as needed.

**Reply by jharrilim** [reply]
Thanks for the reply! I wish I could remember why I had set that to true.

---

Also I realize you are the same person who has to deal with my cancellation messages and I'd like to say I'm sorry
